spyder/widgets/mixins.py

```diff
@@ -755,8 +953,8 @@ def show_object_info(self, text, call=False, force=False):
                             tiptext = name + argspec
                         else:
                             tiptext = signature
-                        self.show_calltip(_("Arguments"), tiptext,
-                                          signature=True, color='#2D62FF')
+                        # TODO: Select language and pass it to call

```

Comment:
Please mention this in the same issue about getting the kernel language from the console. I think both things can be solved in the same PR.

Reply:
https://github.com/spyder-ide/spyder/issues/9194